system-addon/content-src/components/Base/Base.jsx

```diff
@@ -77,10 +77,6 @@ export class _BaseContent extends React.PureComponent {
 
     const outerClassName = `outer-wrapper${shouldBeFixedToTop ? " fixed-to-top" : ""}`;
 
-    if (!props.isPrerendered && !initialized) {
```

Comment:
Can you explain this change?

Reply:
It's not necessary; there is already the exact same guard in the (parent) Base component, so it doesn't need to be in BaseContent also.